paddle/fluid/framework/ir/graph_pattern_detector.h

```diff
@@ -592,12 +592,12 @@ struct FC : public PatternBase {
 // op: fc
 // named node:
 // fc
-// w, bias, output
+// w, bias, output, residual_data
 struct FCMKLDNN : public PatternBase {
   FCMKLDNN(PDPattern* pattern, const std::string& name_scope)
       : PatternBase(pattern, name_scope, "fc_mkldnn") {}
 
-  PDNode* operator()(PDNode* x, bool with_bias);
+  PDNode* operator()(bool with_residual_data);
```

Comment:
with_bias was actually always ignored. I didn't change the behavior of fc with or without bias. Since it was ignored, both with and without bias ops are detected